File: LogicadeProgramacao/Estrutura Sequencial/3. Python/PesoIdealFM.py
altura = float(input("Digite a altura da pessoa: "))
genero = input("Digite o gênero da pessoa (M/F): ")

if genero == "M":
    peso_ideal = (72.7 * altura) - 58
elif genero == "F":
    peso_ideal = (62.1 * altura) - 44.7
else:
    print("Gênero inválido!")
    exit()

print(f"O peso ideal para a altura de {altura:.2f}m e gênero {genero} é: {peso_ideal:.2f}kg")

File: LogicadeProgramacao/Estrutura Sequencial/2. Java/PesoIdealFM.java
import java.util.Scanner;

public class PesoIdealFM {
    public static void main(String[] args) {
        double altura, pesoIdeal;
        char genero;
        Scanner sc = new Scanner(System.in);
        
        System.out.print("Digite a altura da pessoa: ");
        altura = sc.nextDouble();
        
        System.out.print("Digite o gênero da pessoa (M/F): ");
        genero = sc.next().charAt(0);
        
        if (genero == 'M') {
            pesoIdeal = (72.7 * altura) - 58;
        } else if (genero == 'F') {
            pesoIdeal = (62.1 * altura) - 44.7;
        } else {
            System.out.println("Gênero inválido!");
            return;
        }
        
        System.out.printf("O peso ideal para a altura de %.2fm e gênero %s é: %.2fkg", altura, genero, pesoIdeal);
        
        sc.close();
    }
}
